lib/jira_conn.py

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Great!  this looks like boiler plate (I assume provided by confluence), correct?

Reply:
Not really sure about the answer.. I mean the end points are provided and documented in confluence, then we need to adapt a bit the queries to get the pagination and the jql filters. Not sure if that answers your question...